src/components/RequirementHeader.vue

```diff
@@ -1,18 +1,18 @@
 <template>
   <div class="requirementheader">
     <!-- TODO change for multiple colleges -->
-    <div v-if="reqIndex<=2 || reqIndex == 2 + majors.length" class="row top">
+    <div v-if="reqIndex<=1 || reqIndex == 1 + majors.length" class="row top">
```

Comment:
Whether or not we add multiple colleges, this random hardcoded index is definitely not great. Could you define a new variable in data set to 1 and use it in all the places you changed it from 2 to 1? And maybe add a comment about what it does, I'm not certain rn.

Reply:
Will do! Was also not completely sure, but I think it's the # of sections (where each college, major, minor is a section)